pysingfel/experiment/spi.py

```diff
@@ -5,7 +5,7 @@
 
 
 class SPIExperiment(Experiment):
-    def __init__(self, det, beam, particle, orientations=None):
+    def __init__(self, det, beam, jet_radius, particle, orientations=None):
```

Comment:
SPI almost never uses a liquid jet, because the jet will overwhelm any weak signal from sample

aerosol injection is the preferred method. Question is do we need this variable in SPIExperiment?

Reply:
I see. Then `jet_radius` should be input variable for only FXSExperiment and HOLOExperiment?

Reply:
Remove the input parameter  `jet_raduis` in  SPIExperiment class.